### ISSUE TITLE
head_object: error parsing Last-Modified header in 24-hour format

### ISSUE THREAD

**Issue by sa-0001** [taylorfinnell/awscr-s3#93]
the following error occurs, when calling `head_object`:

```
#<Time::Format::Error:Invalid hour for 12-hour clock at 25: "Fri, 26 Mar 2021 16:26:13>>">
```

the value of the Last-Modified header is shown in the error message.  i was able to temporarily fix the problem by overriding either  `HeadObjectOutput.from_response` or `HeadObjectOutput.parse_date`,

**Comment by matthewmcgarvey** [comment]
Fixed by https://github.com/taylorfinnell/awscr-s3/commit/c385112197d8c1b62aca48cd6e196e4f65712893 but not released yet

**Comment by sa-0001** [comment]
i'll keep an eye on it; thank you!

**Comment by taylorfinnell** [comment]
Should be fixed in `0.8.2` now
